src/main.rs

```diff
@@ -726,17 +726,11 @@ impl BFInterp {
             IntPredicate::SLE => is_true = op0 <= op1,
         }
 
-        let is_true_bit: i8;
-        if is_true {
-            is_true_bit = 1;
-        } else {
-            is_true_bit = 0;
-        }
-
-        let is_true_bit = is_true_bit.as_();
+        let is_true: u8 = is_true.into();
```

Comment:
What does `value` expect? `u64`? Can't we just do `value: is_true as u64`?

Reply:
Yes. You're right.